drs_filer/ga4gh/drs/server.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from flask import (current_app)
+from flask.globals import request
+
+from drs_filer.ga4gh.drs.endpoints import registerNewObject
+
+logger = logging.getLogger(__name__)
+
+
+def registerObject():
+    db_collection = (
+        current_app.config['FOCA'].db.dbs['drsStore'].
+        collections['objects'].client
+    )
+    response = registerNewObject.registerNewObject(
+        db_collection,
+        request.json
+    )
+    return response
+
+
+def getObject(object_id):
+    db_collection = (
+        current_app.config['FOCA'].db.dbs['drsStore'].
+        collections['objects'].client
+    )
```

Comment:
If I change `objects` to `object` (spelling mistake), then this results into `KeyError`.

Reply:
Well, if you want `object`, just change it to `object` in the config...

Reply:
I was actually just informing on the error handling part. :smile: